<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scatter</class>
 <widget class="QWidget" name="Scatter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>284</width>
     <height>455</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>预设视角</string>
        </property>
        <property name="buddy">
         <cstring>comboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>FrontLow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Front</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FrontHigh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LeftLow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Left</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LeftHigh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RightLow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Right</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RightHigh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BehindLow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Behind</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BehindHigh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IsometricLeft</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IsometricLeftHigh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IsometricRight</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IsometricRightHigh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DirectlyAbove</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DirectlyAboveCW45</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DirectlyAboveCCW45</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FrontBelow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LeftBelow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RightBelow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BehindBelow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DirectlyBelow</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>水平旋转</string>
        </property>
        <property name="buddy">
         <cstring>sliderH</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="sliderH">
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>垂直旋转</string>
        </property>
        <property name="buddy">
         <cstring>sliderV</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="sliderV">
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>缩  放</string>
        </property>
        <property name="buddy">
         <cstring>sliderZoom</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cBoxTheme">
        <item>
         <property name="text">
          <string>Qt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Primary Colors</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Digia</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>StoneMoss</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ArmyBlue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Retro</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ebony</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Isabelle</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="sliderZoom">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>主  题</string>
        </property>
        <property name="buddy">
         <cstring>cBoxTheme</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="cBoxBarStyle">
        <item>
         <property name="text">
          <string>Bar</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cube</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pyramid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cone</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cylinder</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BevelBar</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BevelCube</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sphere</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Minimal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Arrow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Point</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>散点形状</string>
        </property>
        <property name="buddy">
         <cstring>cBoxBarStyle</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>轴字体大小</string>
        </property>
        <property name="buddy">
         <cstring>spinFontSize</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="spinFontSize">
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>散点大小</string>
        </property>
        <property name="buddy">
         <cstring>spinItemSize</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="spinItemSize">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.020000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnBarColor">
      <property name="text">
       <string>散点的颜色</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QPushButton" name="btnAddPoint">
        <property name="text">
         <string>增加点</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btnReducePoint">
        <property name="text">
         <string>减少点</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnInit">
        <property name="text">
         <string>初始化</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
